rust/platform/triple_mappings.bzl

```diff
@@ -61,6 +65,7 @@ _SYSTEM_TO_DYLIB_EXT = {
     # TODO(acmcarther): To be verified
     "windows": ".dll",
     "emscripten": ".js",
+    "unknown": ".wasm",
```

Comment:
It seems somewhat presumptive to say that any "unknown" system == wasm. I understand that this is a consequence of the current way target triples are resolved, but I find it regrettable.

Reply:
Agreed. Let me see if I can find a better way to do this.

Reply:
I suspect it wont be possible to fix this with the current structuring. Seems the assumption that components of target triples could be independently resolved was a faulty one.

I think we can go with what's here for now with a note that target triples need their properties resolved "holistically" (as in, with the full triple available). I was going to file an issue for this, but it seems https://github.com/bazelbuild/rules_rust/issues/206 captures it pretty well. I'll mirror this comment into that issue for completeness.

Reply:
Makes sense. Then is it okay to leave it like this for now?